content/docs/examples/multicluster/split-horizon-eds/index.md

```diff
@@ -136,11 +136,14 @@ This will be used to access pilot on `cluster1` securely using the ingress gatew
 
     {{< text bash >}}
     $ export LOCAL_GW_ADDR=$(kubectl get --context=$CTX_CLUSTER1 svc --selector=app=istio-ingressgateway \
-        -n istio-system -o jsonpath="{.items[0].status.loadBalancer.ingress[0].ip}")
+        -n istio-system -o jsonpath="{.items[0].status.loadBalancer.ingress[0].ip}") && echo ${LOCAL_GW_ADDR}
     {{< /text >}}
 
-    This command sets the value to the gateway's public IP, but note that you can set it to
-    a DNS name instead, if you have one.
+    This command sets the value to the gateway's public IP and displays it.
+
+    {{< warning >}}
+    The command above only works if the load balancer configuration provides an IP. Support for using DNS names is not yet implemented.
```

Comment:
Follow the international standard for warning text. The markup already gives your the right format and symbol, just focus on the text. This is a great doc to learn more: https://www.mysafetylabels.com/how-to-write-a-warning-label
```suggestion
    The command fails if the load balancer configuration doesn't include an IP address. Ensure the configuration includes the needed IP. The implementation of DNS support is pending. 
```

Reply:
`Ensure the configuration includes the needed IP.`
I feel like this is a bad suggestion, because it might lead people to change the k8s Service definition and thus to possibly break the load balancer.